app/assets/stylesheets/views/_specialist-document.scss

```diff
@@ -1,3 +1,33 @@
+$contents-link-background: #dee0e2;
+
 .specialist-document {
   @include sidebar-with-body;
 }
+
+.govuk-sticky-element {
+  width: 100%;
+  margin: 0 $gutter-half;
+}
+
+.contents-list-link-wrapper {
+  background-color: $contents-link-background;
+  padding: 0 4px 0 20px;
+  width: 100%;
+}
+@media (min-width: 641px) {
+  .contents-list-link-wrapper {
+    width: 66.67%;
+  }
+}
+.app-c-back-to-top {
+  display: block;
+  margin: $gutter-half 0;
+}
+.app-c-back-to-top {
+    display: block;
+    margin: 15px 0 10px 0;
+}
```

Comment:
You have 2 conflicting bit of css here - not sure which margin we want applying. But we can move this to `stylesheets/components/_back-to-top.scss`